### ISSUE TITLE
Video Grid Item: Support clicking image for navigation.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/Learn#17]
Right now the link under the image will lead you to the workshop. You should be able to click the image as well.  I didn't wrap the image within the <a> because I don't think that's great for accessibility. Will revisit.
